packages/genesys-spark-components/src/components/beta/gux-form/gux-form-description/gux-form-description.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component, h, JSX } from '@stencil/core';
+
+/**
+ *  @slot description - Slot for description.
+ */
+
+@Component({
+  styleUrl: 'gux-form-description.scss',
+  tag: 'gux-form-description',
+  shadow: true
+})
+export class GuxFormDescription {
+  render(): JSX.Element {
+    return (<slot name="description"></slot>) as JSX.Element;
```

Comment:
I chose the route of named slots. I dont  think there is many solutions to solve this problem. I also considered the the logging of an error/warning but it would sort of be the same thing as you would want them to slot a element for the styles to apply correctly.  Open to opinions on this.